src/pages/Lend/LoanSearchPage.vue

```diff
@@ -240,6 +254,12 @@ export default {
 				this.challengeData = goalsData?.goals?.values?.[0] || {};
 				const teamData = this.apollo.readQuery({ query: TeamInfoFromId, variables: { team_id: teamId } });
 				this.teamData = teamData?.community?.team || {};
+
+				const loanIds = this.challengeData?.targets?.values?.map(target => target?.loanId) ?? [];
+
+				loanIds.forEach(loanId => {
```

Comment:
I think we just need add loan ids when some of them are added to the basket successfully...

Reply:
Exactly. I don't think the changes in this method are needed, just the changes to `addLoanToChallengeCookie`